src/webui/www/private/views/preferences.html

```diff
@@ -1702,6 +1707,19 @@
             $('mail_password_text').setProperty('disabled', !isMailAuthEnabled);
         };
 
+        const sendTestEmail = function() {
+            new Request({
+                url: 'api/v2/app/sendTestEmail',
+                method: 'post',
```

Comment:
Is it intended to be `POST` method only? If so, please add entry here: https://github.com/qbittorrent/qBittorrent/blob/dc501c39bee1c0bf8ecea66e9e6c377727b33c33/src/webui/webapplication.h#L154-L155